docs/source/checks/tabular/model_evaluation/plot_calibration_score.py

```diff
@@ -4,9 +4,30 @@
 
 Calibration Score
 *****************
+This notebook provides an overview for using and understanding the Calibration Score check.
+
+**Structure:**
+
+* `What is the Calibration Score check? <#what-is-the-calibration-score-check>`__
+* `Binary Classification <#binary-classification>`__
+* `Multi-class classification <#multi-class-classification>`__
+
+
+What is the Calibration Score check?
```

Comment:
Great explanation!

Reply:
Would maybe add what actually calibration is - that the output of the model (for this class) directly represents the probability of this class to be the correct answer (the label). I don't think this is clear for everyone. You can also give an example ("e.g. a probability of 0.7 for class A represents that there is 70% chance the true label of this sample is actually class A")